tutorials/notebooks/keras/qat/example_keras_qat.py

```diff
@@ -27,51 +27,92 @@
 import numpy as np
 import tensorflow as tf
 from keras import Model, layers, datasets
-from keras.datasets import mnist
+
+from model_compression_toolkit import DefaultDict
+from model_compression_toolkit.target_platform_capabilities.target_platform.op_quantization_config import AttributeQuantizationConfig
+from model_compression_toolkit.constants import FLOAT_BITWIDTH
+from model_compression_toolkit.target_platform_capabilities.constants import KERNEL_ATTR, KERAS_KERNEL, BIAS_ATTR, BIAS
 
 import model_compression_toolkit as mct
 import tempfile
 
+tp = mct.target_platform
+
 
 def get_tpc():
     """
-    Assuming a target hardware that uses power-of-2 thresholds and quantizes weights and activations
-    to 2 and 3 bits, accordingly. Our assumed hardware does not require quantization of some layers
+    Assuming a target hardware that uses a power-of-2 threshold for activations and
+    a symmetric threshold for the weights. The activations are quantized to 3 bits, and the weights
```

Comment:
Weights are in 2-bit in this example, not 3 (also fix in notebook).
Also, maybe say that "the kernel weights are quantized to 2 bits", instead of "the weights are quantized..."